src/components/shop/menu.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @module components/shop
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { getSafeArea } from "./shop-layout.js";
+import { createGelButtons, resizeGelButtons } from "./menu-buttons.js";
+
+export const createMenu = (scene, config) => {
+    const bounds = getSafeArea(scene.layout);
```

Comment:
there's two getSafeArea function calls in two shop files, maybe its better to call it once from the outside and these functions just get passed the area to use?

Reply:
I was able to eliminate a few of these getSafeArea calls: 3d2ba13